src/main/kotlin/com/hibob/academy/dao/PetDao.kt

```diff
@@ -0,0 +1,88 @@
+package com.hibob.academy.dao
+
+import org.jooq.Record
+import org.jooq.DSLContext
+import org.jooq.RecordMapper
+
+
+class PetDao(private val sql: DSLContext) {
+    private val pet = PetTable.instance
+    private val petDataMapper= RecordMapper<Record, PetData> { record ->
+        PetData(
+            record[pet.petName],
+            record[pet.dateOfArrival],
+            record[pet.companyId],
```

Comment:
Does pet need to have an id?
What about ownerId, do we want pet to have this or not?